waltz-server/src/main/java/com/wepay/waltz/server/WaltzServer.java

```diff
@@ -352,6 +353,10 @@ public boolean isClosed() {
         return partitionIds;
     }
 
+    public NetworkServer getNetworkServer() {
```

Comment:
I don't want to expose `NetworkServer`. Can you change it to `closeNetworkServer()`?

Reply:
will fix.